docs/tutorial/updates.md

```diff
@@ -261,7 +261,49 @@ server-communication aspect of the process by loading your update from a local d
 
 ## Update server specification
 
-A Squirrel-compatible update server has different
+For advanced deployment needs, you can also roll out your own Squirrel-compatible update server.
+Squirrel.Windows and Squirrel.Mac clients require different response formats,
+but you can use a single server for both platforms by sending requests to
+different endpoints depending on the value of `process.platform`.
+
+```js title='main.js'
+const { app, autoUpdater } = require('electron')
+
+const server = 'https://your-deployment-url.com'
+// e.g. for Windows and app version 1.2.3
+// https://your-deployment-url.com/update/win32/1.2.3
+const url = `${server}/update/${process.platform}/${app.getVersion()}`
+
+autoUpdater.setFeedURL({ url })
+```
+
+### Windows
```

Comment:
We should note that it's expected that your RELEASES file gets bigger with each release, in it's final form it effectively lists all your releases even though only the latest one matters

Reply:
Can you elaborate on that point? The one we have set up for Fiddle only shows the latest release:

* https://github.com/electron/fiddle/releases/tag/v0.36.4
* https://update.electronjs.org/electron/fiddle/win32-x64/0.36.0/RELEASES